config/filesystems.php

```diff
@@ -47,6 +47,7 @@
 			'root' => env('LYCHEE_UPLOADS', public_path('uploads/')),
 			'url' => env('LYCHEE_UPLOADS_URL', 'uploads/'),
 			'visibility' => env('LYCHEE_IMAGE_VISIBILITY', 'public'),
+			'directory_visibility' => env('LYCHEE_IMAGE_VISIBILITY', 'public'),
```

Comment:
wouldn't we want this to be in the `.env.example` too ? :thinking:

Reply:
I don't understand what you mean. This PR adds a missing entry to the configuration file. The environment variable `LYCHEE_IMAGE_VISIBILITY` is the same as before. So there is nothing to change with respect to `.env.example`.

Reply:
Oh, my mistake, I thought it was a different value than for `visibility`.